docs/storage-abstraction.md

```diff
@@ -751,45 +806,42 @@ Delete delete =
         .clusteringKey(clusteringKeyForDelete)
         .build();
 
-// Execute the operations
+// Execute the operations.
 storage.mutate(Arrays.asList(put, delete));
 ```
 
-Note that a Mutate operation only accepts mutations for a single partition; otherwise, it throws an exception.
+{% capture notice--info %}
+**Note**
+
+A Mutate operation only accepts mutations for a single partition; otherwise, an exception will be thrown.
+
+In addition, if you specify multiple conditions in a Mutate operation, the operation will be executed only when all the conditions match.
+{% endcapture %}
 
-And if you specify multiple conditions in a Mutate operation, the operation is executed only when all the conditions match.
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-#### Use a default namespace for CRUD operations
+#### Default namespace for CRUD operations
```

Comment:
I assume this PR should be applied to versions 3.5 and later, correct? Also, as you know, this namespace config was introduced in version 3.9 and later. So, please remove it when applying to versions 3.8 and earlier.

Reply:
Got it. I've noted this and will remove it from 3.8 and earlier versions.